guides/logging.txt

```diff
@@ -0,0 +1,24 @@
+TOPIC
+    logging
+
+DESCRIPTION
+    Bolt logs messages both to the console and to log files. Messages can
+    either come from Bolt's internal 'outputter' class, which logs user-facing
+    messages like progress and results, or from the 'logger' class, which logs
+    warnings, errors, and log-structured output to log files. Both of these message
+    streams are configurable.
+
+    By default, Bolt logs to the console at 'warn' level, writes a log file to
+    '<project>/bolt-debug.log' at 'debug' level, and runs with the outputter in
+    verbose mode unless you are running a plan, in which case it runs without
+    verbose (you still with me?). You can configure logging output in your
+    project configuration, and can configure console log levels (but not file log
+    levels) with `--log-level`. For example, we recommend running `--log-level
+    debug` to quickly debug things like connection issues and unexpected run
+    results. You can configure the user-facing output only on the command line with
```

Comment:
I think we discussed moving towards encouraging users to check `bolt-debug.log` over running `--log-level debug`, since users wouldn't need to run a command again in debug mode.

Reply:
Oh hm yeah....I think I'll just remove that line, I'm not sure what else to recommend (recommending using `--log-level error` to silence warnings seems....not great.